test/integration/BarbellTest.cpp

```diff
@@ -0,0 +1,282 @@
+
+        // This map: _endpointNetworkLinkMap is populated automatically on endpoint creation.
+        // Following it's evolution allows to find Endpoint/NetworkLink for paritcular component.
+        _endpointNetworkLinkMap.clear();
+        auto sdp1 = bb1.allocate(baseUrl, conf.getId(), true);
+        auto interBridgeEndpoints1 = _endpointNetworkLinkMap;
+
+        auto sdp2 = bb2.allocate(baseUrl2, conf2.getId(), false);
+
+        bb1.configure(sdp2);
+        bb2.configure(sdp1);
+
+        for (const auto& linkInfo : interBridgeEndpoints1)
+        {
+            linkInfo.second.ptrLink->setLossRate(PACKET_LOSS_RATE);
+        }
+
+        utils::Time::nanoSleep(2 * utils::Time::sec);
+
+        group.clients[0]->initiateCall(baseUrl, conf.getId(), true, true, true, true);
+        group.clients[1]->initiateCall(baseUrl2, conf2.getId(), false, true, true, true);
+        group.clients[2]->initiateCall(baseUrl2, conf2.getId(), false, true, true, true);
+
+        ASSERT_TRUE(group.connectAll(utils::Time::sec * 5));
+
+        make5secCallWithDefaultAudioProfile(group);
+
+        HttpGetRequest statsRequest((std::string(baseUrl) + "/stats").c_str());
+        statsRequest.awaitResponse(1500 * utils::Time::ms);
+        EXPECT_TRUE(statsRequest.isSuccess());
+        HttpGetRequest confRequest((std::string(baseUrl) + "/conferences").c_str());
+        confRequest.awaitResponse(500 * utils::Time::ms);
+        EXPECT_TRUE(confRequest.isSuccess());
+
+        bb1.remove(baseUrl);
+
+        utils::Time::nanoSleep(utils::Time::ms * 1000); // let pending packets be sent and received)
+
+        group.clients[0]->_transport->stop();
+        group.clients[1]->_transport->stop();
+        group.clients[2]->_transport->stop();
+
+        group.awaitPendingJobs(utils::Time::sec * 4);
+        finalizeSimulation();
+
+        logVideoSent("client1", *group.clients[0]);
+        logVideoSent("client2", *group.clients[1]);
+        logVideoSent("client3", *group.clients[2]);
+
+        {
+            auto videoCounters = group.clients[0]->_transport->getCumulativeVideoReceiveCounters();
+            EXPECT_EQ(videoCounters.lostPackets, 0);
+
+            auto autioCounters = group.clients[0]->_transport->getCumulativeAudioReceiveCounters();
+            EXPECT_NE(autioCounters.lostPackets, 0);
+
+            const auto& rData1 = group.clients[0]->getAudioReceiveStats();
+            std::vector<double> allFreq;
+            EXPECT_EQ(rData1.size(), 2);
+
+            for (const auto& item : rData1)
+            {
+                if (group.clients[0]->isRemoteVideoSsrc(item.first))
+                {
+                    continue;
+                }
+
+                std::vector<double> freqVector;
+                std::vector<std::pair<uint64_t, double>> amplitudeProfile;
+                auto rec = item.second->getRecording();
+                analyzeRecording(rec, freqVector, amplitudeProfile, item.second->getLoggableId().c_str());
```

Comment:
@olofkallander, @RicardoMDomingues, do you think we need this audio analisys in this kind of test? It takes some time (~5 s) to analyze, making overall test execution slower. But in the same time, it guarantees that the data we receive is intact.

Reply:
imo 5sec is not a big deal. But I guess testing if the audio is intact is only necessary for participants with mixed audio where smb needs to decode, mix and encode again, otherwise as sfu we are interested only in understanding if we forwarder the packets as we should and we should be able do that without analyse the audio it self

Reply:
imo 5sec is not a big deal. But I guess testing if the audio is intact is only necessary for participants with mixed audio where smb needs to decode, mix and encode again, otherwise as sfu we are interested only in understanding if we forwarder the packets as we should and we should be able do that without analyse the audio it self

Reply:
OK. I'll keep it here. But I'll skip it in the next tests where we'll check packet losses and video recovery.